artemis-server/src/main/java/org/apache/activemq/artemis/core/management/impl/ActiveMQServerControlImpl.java

```diff
@@ -952,6 +952,7 @@ public void deployQueue(final String address,
       clearIO();
       try {
          server.createQueue(new QueueConfiguration(name)
+                            .setAddress(address)
                             .setName(name)
```

Comment:
You can actually remove `setName(name)` as this is already taken care of by passing `name` to the constructor. The bug here is that `setName(name)` is actually called instead of `setAddress(address)`.